src/flows/pipes/Notification/Expressions.tsx

```diff
@@ -0,0 +1,220 @@
+import React, {FC, useState, useMemo, useCallback, useContext} from 'react'
+
+import {
+  Input,
+  InputType,
+  IconFont,
+  DapperScrollbars,
+  EmptyState,
+  ComponentSize,
+} from '@influxdata/clockface'
+import {FromFluxResult} from '@influxdata/giraffe'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import Expression from 'src/flows/pipes/Notification/Expression'
+import {FlowContext} from 'src/flows/context/flow.current'
+
+interface Props {
+  parsed: FromFluxResult
+  id: string
+}
+
+interface Schemas {
+  [key: string]: SchemaExpressions
+}
+
+interface SchemaExpressions {
+  [key: string]: Array<string>
+}
+
+const Expressions: FC<Props> = ({id, parsed}) => {
+  const [search, setSearch] = useState('')
+  const {flow, updateData} = useContext(FlowContext)
+  const updateSearch = useCallback(
+    e => {
+      setSearch(e.target.value)
+    },
+    [search, setSearch]
+  )
+
+  const parsedResultToSchema = (parsed: FromFluxResult): Schemas => {
+    let ni
+    const filtered = [
+      /^_start$/,
+      /^_stop$/,
+      /^_time$/,
+      /^_value/,
+      /^_measurement$/,
+      /^_field$/,
+      /^table$/,
+      /^result$/,
+    ]
+    if (!parsed) {
+      return
+    }
+
+    const out = parsed.table as any
+    const len = out.length
+    const measurements = out.columns._measurement?.data
+    const fields = out.columns._field?.data
+    const columns = out.columnKeys.filter(key => {
+      return filtered.reduce((acc, curr) => {
+        return acc && !curr.test(key)
+      }, true)
+    })
+    const schema = {} as any
+
+    for (ni = 0; ni < len; ni++) {
+      if (!schema.hasOwnProperty(measurements[ni])) {
+        schema[measurements[ni]] = {}
+      }
+
+      // TODO: is there always only one field?
+      const filteredFields = [fields[ni]].filter(f =>
+        f.toLowerCase().includes(search.toLowerCase())
+      )
+      if (filteredFields.length) {
+        schema[measurements[ni]].fields = filteredFields
+      }
+
+      const filteredMeasurements = [
+        ...new Set<string>(measurements),
+      ].filter(m => m.toLowerCase().includes(search.toLowerCase()))
+      if (filteredMeasurements.length) {
+        schema[measurements[ni]].measurements = filteredMeasurements
+      }
+
+      const filteredSystems = [
+        '_check_id',
+        '_check_name',
+        '_level',
+        '_source_measurement',
+        '_type',
+      ].filter(s => s.toLowerCase().includes(search.toLowerCase()))
```

Comment:
Russ didn't know of a way to dynamically populate these. So that's why they are hard coded here